i18n/locale/no/datebox.po

```diff
@@ -285,8 +285,8 @@ msgstr "false"
 
 #. Set to the start day of the calendar (0=Sunday)
 #: generate-i18n.py:42
-msgid "0"
```

Comment:
Should this be set to 0, as it translates from the default locale to Norwegian locale?